source/Cute/Commands/Content/ContentJoinCommand.cs

```diff
@@ -0,0 +1,165 @@
+﻿using Contentful.Core.Errors;
+using Contentful.Core.Models;
+using Contentful.Core.Models.Management;
+using Contentful.Core.Search;
+using Cute.Commands.BaseCommands;
+using Cute.Commands.Login;
+using Cute.Config;
+using Cute.Lib.Contentful;
+using Cute.Lib.Contentful.CommandModels.ContentJoinCommand;
+using Cute.Lib.Exceptions;
+using Cute.Lib.Serializers;
+using Cute.Services;
+using Newtonsoft.Json.Linq;
+using Spectre.Console;
+using Spectre.Console.Cli;
+using System.ComponentModel;
+
+namespace Cute.Commands.Content;
+
+public class ContentJoinCommand(IConsoleWriter console, ILogger<ContentJoinCommand> logger, ContentfulConnection contentfulConnection,
+    AppSettings appSettings) : BaseLoggedInCommand<ContentJoinCommand.Settings>(console, logger, contentfulConnection, appSettings)
+{
+    public class Settings : LoggedInSettings
+    {
+        [CommandOption("-i|--join-id")]
```

Comment:
use [CommandOption("-k|--key")] for this parameter for consistency with sync api and generate commands